src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public final class FigureSupplier {
+    public Figure getRandomFigure() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        int indexFigure = new Random().nextInt(5);
+        switch (indexFigure) {
+            case 0:
+                return new Circle(colorSupplier.getRandomColor(),getInt());
+            case 1:
+                return new IsoscelesTrapezoid(colorSupplier.getRandomColor(),
+                        getInt(),
+                        getInt(),
+                        getInt());
```

Comment:
create a separate method for each of the figure that will return random specific figure, for example, getRandomCircle()